src/main/cpp/t04_both2.cpp

```diff
@@ -23,9 +23,32 @@
 
 #include "t04_both2.h"
 #include <iostream>
-
+#include <set>
 using namespace std;
 
 int t04_both2() {
+	int n, k, m, c = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'c' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4676905)